src/zmalloc.c

```diff
@@ -55,6 +55,8 @@ void zlibc_free(void *ptr) {
 #include "zmalloc.h"
 #include "atomicvar.h"
 
+#define UNUSED(x) (void)(x)
```

Comment:
This line in REDIS_TEST is redefined.
https://github.com/redis/redis/blob/815a6f846a3836b87a4601e5ce6152d8d534dce1/src/zmalloc.c#L751

Reply:
thanks. fixed.